wolfcrypt/src/cmac.c

```diff
@@ -56,6 +56,46 @@
     #include <wolfssl/wolfcrypt/cryptocb.h>
 #endif
 
+#ifdef WOLFSSL_HASH_KEEP
+/* Some hardware have issues with update, this function stores the data to be
+ * hashed into an array. Once ready, the Final operation is called on all of the
+ * data to be hashed at once.
+ * returns 0 on success
+ */
+static int _wc_CMAC_Grow(byte** msg, word32* used, word32* len, const byte* in,
```

Comment:
Is it possible to use your new `_wc_Sha_Grow` and rename it? Also if not please use XREALLOC below.